File: act4/classes/People/Person.php
<?php

class Person
{
	public $max_health=100;
	public $health=100;
	public $name='';
	public $food=0;
	function __construct($name)
	{
		$this->name=$name;
	}
	/*function eat(food):
	{
		self.health=self.health+food
		if (self.health>=max_health):
			self.health=max_health
		if (self.health<=0):
			self.health=0
	}*/
	public function is_alive()
	{
		if($this->health > 0)
		{
		    return True;
		}
		else
			return False;
	}
}

?>

File: act2/People/Person.py
class Person():
	health=100
	name=''
	food=0
	max_health=100
	def __init__(self,name,health):
		self.health=health
		self.name=name
	def eat(self,food,max_health):
		self.health=self.health+food
		if (self.health>=max_health):
			self.health=max_health
		if (self.health<=0):
			self.health=0

